GitCommands/Worktrees/WorktreeInfo.cs

```diff
@@ -0,0 +1,35 @@
+namespace GitCommands.Worktrees
```

Comment:
I decided to create new functionality for picograms and leave old functionality for this form because it should be refactored. I'm going to do it later during the fixing of #8696